jest.config.js

```diff
@@ -1,11 +1,14 @@
-module.exports = {
+export default {
   testPathIgnorePatterns: ['<rootDir>/e2e/'],
   setupFiles: ['<rootDir>/_jestsetup.js'],
   moduleNameMapper: {
-    '\\.css$': '<rootDir>/__mocks__/styleMock.js',
+    '\\.css$': 'identity-obj-proxy',
```

Comment:
The old setup wasn't working and this is what they had in the docs for doing css imports.